app/models/cdo/manage/cdo.js

```diff
@@ -62,8 +64,13 @@ function ViewModel (details, cdo, backNav, continueLink) {
           }
         }
 
+        let queryParams = ''
+        if (task === tasks.verificationDateRecorded) {
+          queryParams += '&clear=true'
+        }
+
         if (status !== 'Cannot start yet') {
-          taskProperties.href = `/cdo/manage/task/${key}/${modelDetails.dogIndex}${backNav.srcHashParam}`
+          taskProperties.href = `/cdo/manage/task/${key}/${modelDetails.dogIndex}${backNav.srcHashParam}${queryParams}`
```

Comment:
Can srcHashParams be blank or undefined here? If so, the url will fail because no '?'